src/sentry/tasks/servicehooks.py

```diff
@@ -1,13 +1,98 @@
 from __future__ import absolute_import, print_function
 
 import six
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
 from time import time
 
+from sentry.api.serializers import serialize, app_platform_event
 from sentry.http import safe_urlopen
+from sentry.models import Group, SentryAppInstallation, ServiceHook
 from sentry.tasks.base import instrumented_task
 from sentry.utils import json
 from sentry.utils.http import absolute_uri
 
+# This is an extra, explicit, measure to ensure we only send events for
+# resource changes we deem necessary.
+ALLOWED_ACTIONS = (
+    'issue.created',
+)
+
+# We call some models by a different name, publically, than their class name.
+# For example the model Group is called "Issue" in the UI. We want the Service
+# Hook events to match what we externally call these primitives.
+RESOURCE_RENAMES = {
+    'group': 'issue',
+}
+
+
+@receiver(post_save, sender=Group)
+def resource_changed(sender, instance, created, **kwargs):
+    process_resource_change.delay(sender, instance.id, created)
+
+
+@instrumented_task('sentry.tasks.process_resource_change')
```

Comment:
Should you be retrying on network related failures?

Reply:
Yea, I keep thinking `@instrumented_task` has sane defaults for retries, but you actually have to decorate with `@retry` as well. Will do 👍